src/main/graphql-server/middleware.js

```diff
@@ -15,6 +18,48 @@ const btoa = arg => {
 const Authorization = 'Basic ' + btoa('admin:admin')
 
 const router = express.Router()
+
+const rootLogger = createLogger({
+  level: 'info',
+  format: format.combine(
+    format.timestamp(),
+    format.errors({ stack: true }),
+    format.json()
+  ),
+  defaultMeta: { service: 'revelio' },
+  transports: [
+    process.env.NODE_ENV === 'production'
+      ? new transports.Console()
```

Comment:
I think for dev, because of the webpack dev server, we should write to a file. For prod, I think it's pretty standard to use stdout and let someone else deal with where the logs go.